### PR TITLE
Heat Sources Update

### PR DESCRIPTION
**Pull request by MuteTiefling**
It feels odd that a piece of coral should be the best heat source here... replaced fire coral with nitro block and added a few more thematic blocks as heat sources.

### CODE REVIEW

File: config/powah/general_common.toml [hunk 1]
```diff
@@ -31,10 +31,11 @@ lens_of_ender = true
 	#Less number more cold, min = -100, max = 1
 	coolantFluids = ["minecraft:water=1"]
 	#
+
 	#List of heat source blocks used under Thermo Generator.
 	#Block registry name = heat, eg: ["minecraft:lava=1000", "minecraft:magma_block=800"]
 	#Min = 1, max = 900000000
-	heatSources = ["minecraft:lava=1000", "minecraft:magma_block=1200", "minecraft:fire_coral_block=8000", "powah:blazing_crystal_block=2800"]
+	heatSources = ["minecraft:lava=1000", "minecraft:magma_block=1200", "powah:nitro_crystal_block=8000", "powah:blazing_crystal_block=2800", "botania:blaze_block"=1600, "botania:blaze_lantern"=1600, "quark:brimstone=1200"]
```
Comment (Quezler) [inline config/powah/general_common.toml:38]:
```suggestion
	heatSources = ["minecraft:lava=1000", "minecraft:magma_block=1200", "powah:nitro_crystal_block=8000", "powah:blazing_crystal_block=2800", "botania:blaze_block=1600", "botania:blaze_lantern=1600", "quark:brimstone=1200"]

```

**Comment by NillerMedDild** [comment]
Aww, I thought I was clever adding the Fire Coral :P but good changes, thank you! I will add the Fire Coral back though, just with a way lower value.

**Comment by MuteTiefling** [comment]
Aww, sorry. Didn't mean to step on your toes there. :(

**Comment by NillerMedDild** [comment]
No worries :P

**Pull request merged**